assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/styles.scss

```diff
@@ -0,0 +1,59 @@
+$breakpoints: (
+	mobile: 600px,
+	tablet: 782px,
+	laptop: 1240px,
+	desktop-lg: 1800px,
+);
+
+@mixin breakpoint($breakpoint, $direction) {
+	@if map-has-key($breakpoints, $breakpoint) {
+		// Get the breakpoint value.
+		$breakpoint-value: map-get($breakpoints, $breakpoint);
+
+		@if $direction == max {
+			@media (max-width: ($breakpoint-value - 1)) {
+				@content;
+			}
+		} @else if $direction == min {
+			@media (min-width: $breakpoint-value) {
+				@content;
+			}
+		} @else {
+			@media ($direction: $breakpoint-value) {
+				@content;
+			}
+		}
+
+		// If the breakpoint doesn't exist in the map.
+	} @else {
+		@if $direction == max {
+			@media (max-width: $breakpoint) {
+				@content;
+			}
+		} @else if $direction == min {
+			@media (min-width: $breakpoint) {
+				@content;
+			}
+		} @else {
+			@media ($direction: $breakpoint) {
+				@content;
+			}
+		}
+	}
+}
```

Comment:
source of the mixin: https://rimdev.io/making-media-query-mixins-with-sass/

Reply:
@tn3rb if you think we might need this, please suggest and appropriate location

Reply:
Moved into mixins folder, in suggested place.
I noticed the same approach here: https://github.com/twbs/bootstrap/tree/master/scss/mixins